lib/buffer.js

```diff
@@ -1,13 +1,18 @@
+/* eslint-disable no-class-assign */
 /* eslint-disable require-buffer */
 'use strict';
 
 const binding = process.binding('buffer');
 const { isArrayBuffer } = process.binding('util');
+const { deprecate } = require('internal/util');
 const bindingObj = {};
 
-class FastBuffer extends Uint8Array {}
+class Buffer extends Uint8Array {}
+
+// to avoid code churn...
+const FastBuffer = Buffer;
+Buffer = DummyBuffer;
```

Comment:
Hm. If the name `DummyBuffer` is just chosen to avoid clashing with the already-existing `Buffer` identifier, maybe one could resolve that by turning the above block into

```js
const FastBuffer = (() => class Buffer extends Uint8Array {})();
```

… not sure if that’s more readable or not.